iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java

```diff
@@ -917,6 +925,9 @@ public void stop() {
       if (dataRegionConsensusStarted) {
         DataRegionConsensusImpl.getInstance().stop();
       }
+      /*if (IoTDBRestServiceDescriptor.getInstance().getConfig().isEnableRestService()) {
```

Comment:
seems we can remove this line? or just put this uncommented line to `deactivate()`?